src/python-common/setup.py

```diff
@@ -0,0 +1,28 @@
+from setuptools import setup, find_packages
+
+
+setup(
+    name='ceph',
+    version='1.0.0',
+    packages=find_packages(),
+    author='',
+    author_email='dev@ceph.io',
+    description='Ceph common Python code.',
```

Comment:
I would like more something like: "Ceph Python Common Framework"

Reply:
This is more like a library and less like a framework. I'm open fur suggestions.